File: app/src/main/java/br/com/alexandre/guide/review/respository/ReviewResponse.java
package br.com.alexandre.guide.review.respository;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class ReviewResponse {

    @SerializedName("status")
    @Expose
    private boolean status;
    @SerializedName("total_reviews_comments")
    @Expose
    private int totalReviewsComments;
    @SerializedName("data")
    @Expose
    private List<Review> review;

    public Boolean getStatus() {
        return status;
    }

    public void setStatus(boolean status) {
        this.status = status;
    }

    public int getTotalReviewsComments() {
        return totalReviewsComments;
    }

    public void setTotalReviewsComments(int totalReviewsComments) {
        this.totalReviewsComments = totalReviewsComments;
    }

    public List<Review> getData() {
        return review;
    }

    public void setData(List<Review> review) {
        this.review = review;
    }
}

File: app/src/main/java/br/com/alexandre/guide/review/repository/ReviewResponse.kt
package br.com.alexandre.guide.review.repository

import com.google.gson.annotations.SerializedName

class ReviewResponse {

    @SerializedName("status")
    val isStatus: Boolean = false
    @SerializedName("total_reviews_comments")
    val totalReviewsComments: Int = 0
    @SerializedName("data")
    val data: List<Review>? = null
}